stubs/cryptography/@tests/stubtest_allowlist.txt

```diff
@@ -18,22 +21,18 @@ cryptography.hazmat.primitives.hmac.HMAC.__init__
 cryptography.hazmat.primitives.hmac.HMAC.update
 cryptography.hazmat.primitives.serialization.pkcs12.serialize_key_and_certificates
 cryptography.hazmat.primitives.serialization.pkcs7.PKCS7SignatureBuilder.__init__
-cryptography.x509.CertificateBuilder.add_extension
 cryptography.x509.CertificateBuilder.public_key
```

Comment:
Are the changes in this file needed because the allowlist was created on different cryptography version?

Reply:
I have no idea. But we should probably only run stubtest for third-party packages that were actually changed.